src/openrct2/world/Park.cpp

```diff
@@ -516,7 +517,12 @@ money32 Park::CalculateRideValue(const Ride* ride) const
 
 money32 Park::CalculateCompanyValue() const
 {
-    return finance_get_current_cash() + gParkValue - gBankLoan;
+    money32 result = gParkValue - gBankLoan;
```

Comment:
Are these small enough to not use `add_clamp_money32`?

Reply:
I dug through the code and concluded that `gBankLoan` is clamped by `gMaxBankLoan`, which is set to `gMaxBankLoan = 5000000` so that is small enough. `gParkValue` is calculated through the `CalculateParkValue()` method:

```
money32 Park::CalculateParkValue() const
{
    // Sum ride values
    money32 result = 0;
    for (const auto& ride : GetRideManager())
    {
        result += CalculateRideValue(&ride);
    }

    // +7.00 per guest
    result += gNumGuestsInPark * MONEY(7, 00);

    return result;
}
```

There is no overflow protection/clamping present, so theoretically it could overflow if there is a massive amount of rides and guest. I don't think that is a realistic scenario, but then again, getting max cash is also a pretty unrealistic scenario and it still drove me to make this fix.